packages/coinstac-ui/app/render/components/computation-submission.jsx

```diff
@@ -102,14 +106,14 @@ ComputationSubmission.propTypes = {
 };
 
 // http://dev.apollodata.com/react/cache-updates.html
-export default graphql(addComputationMetadata, {
+export default graphql(addComputationFunc, {
   props: ({ mutate }) => ({
     submitSchema: computationSchema => mutate({
       variables: { computationSchema },
       update: (store, { data: { addComputation } }) => {
-        const data = store.readQuery({ query: fetchComputationMetadata });
+        const data = store.readQuery({ query: fetchAllComputationsMetadataFunc });
         data.fetchAllComputations.push(addComputation);
-        store.writeQuery({ query: fetchComputationMetadata, data });
+        store.writeQuery({ query: fetchAllComputationsMetadataFunc, data });
```

Comment:
It's not necessary, no. However, things get confusing pretty quickly when you're juggling function names in GraphQL versus the function names that are responsible for calling those GraphQL functions from React. We've appended 'Func' to the end of the React functions to make the distinction clearer. Also, they are used together and would cause conflicts. For instance, here `addComputationFunc` is a React function that is responsible for calling the GraphQL `addComputation`.  `addComputation` returns the newly added computation and we push that computation into `fetchAllComputationsFunc`, which is a query existing in local cache that holds all existing computations. If we didn't append `Func` to the end of the former, there'd be namespace issues with the graphql mutation. [Here's an example from the docs](http://dev.apollodata.com/react/cache-updates.html#directAccess). One thing I am noticing is that they use all caps and append `MUTATION` rather than `Func`, so I'll go through and rename things to match this.